Cython/Utility/ObjectHandling.c

```diff
@@ -1024,6 +1024,91 @@ static PyObject *__Pyx_CreateClass(PyObject *bases, PyObject *dict, PyObject *na
     return result;
 }
 
+/////////////// Py3UpdateBases.proto ///////////////
+
+static PyObject* __Pyx_PEP560update_bases(PyObject *bases); /* proto */
```

Comment:
I'd rather add another underscore to make this name easier to read: `__Pyx_PEP560_update_bases`.